src/index.js

```diff
@@ -57,20 +58,7 @@ function setSelection(el, selection) {
   catch (e) { /* not focused or not visible */ }
 }
 
-var MaskedInput = React.createClass({
-  propTypes: {
-    mask: React.PropTypes.string.isRequired,
-
-    formatCharacters: React.PropTypes.object,
-    placeholderChar: React.PropTypes.string
-  },
-
-  getDefaultProps() {
-    return {
-      value: ''
-    }
-  },
-
+class MaskedInput extends React.Component {
```

Comment:
Isn't nwb generating a UMD and a modules build? Doesn't this already get transformed in the UMD build?